std/assembly/error.ts

```diff
@@ -1,12 +1,11 @@
 export class Error {
 
   name: string = "Error";
-  message: string;
-  stack: string = ""; // TODO
+  stack: string | null = null; // TODO
```

Comment:
Afaik this is never `null` in JS

Reply:
Yes, but it could be undefined according to [this](https://github.com/Microsoft/TypeScript/blob/f53a27eac0ee702bd14d984deaecadfb42181c99/src/lib/es5.d.ts#L945)

Or not?

Reply:
Actually `Error.prototype.stack` is not standard but all browsers support it

Reply:
My initial though was that if it's guaranteed to be a string, even if empty, one can simply print it without having to check explicitly, which is about what you'd do in JS because it's effectively always there.